src/domain/event/__tests__/EditEventPage.test.tsx

```diff
@@ -430,23 +430,31 @@ test('should show server errors', async () => {
   screen.getByText(/lopetusaika ei voi olla menneisyydessä./i);
 });
 
-test('should render fields for external user', async () => {
-  const mocks = [
-    mockedEventResponse,
-    mockedEventTimeResponse,
-    mockedImageResponse,
-    mockedKeywordSelectorKeywordsResponse,
-    mockedAudienceKeywordSetResponse,
-    mockedTopicsKeywordSetResponse,
-    mockedLanguagesResponse,
-    mockedPlaceResponse,
-    mockedPlacesResponse,
-    mockedFilteredPlacesResponse,
-    mockedUserWithoutOrganizationsResponse,
-    mockedOrganizationResponse,
-    mockedOrganizationAncestorsResponse,
+test('should render external user contact fields for admin', async () => {
+  renderComponent();
+
+  await loadingSpinnerIsNotInDocument();
+
+  const externalUserContactFields = [
+    /nimi/i,
+    /sähköpostiosoite/i,
+    /puhelinnumero/i,
+    /organisaatio/i,
+    /olen lukenut tietosuojaselosteen ja annan luvan tietojeni käyttöön/i,
   ];
 
+  const fieldset = await screen.findByTestId('fields-external-user-contact');
```

Comment:
You can avoid using test-id with query:
```suggestion
  const fieldset = await screen.findByRole('group', { name: /yhteystiedot/i });
```
With `findByRole` you also can test that external user section heading findable with e.g. screen readers

Reply:
https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/251/commits/13d1304823525678db1873e17506fc984531e410